libmscore/note.cpp

```diff
@@ -332,6 +364,19 @@ QString NoteHead::type2name(NoteHead::Type type)
       return noteHeadTypeNames[int(type) + 1].name;
       }
 
+//---------------------------------------------------------
+//   name2scheme
+//---------------------------------------------------------
+
+NoteHead::Scheme NoteHead::name2scheme(QString s)
```

Comment:
Add `const` at the end of the method signature

Reply:
There is no `const` [here](https://github.com/mattmcclinch/MuseScore/blob/bd64822d8fb3e7f18dd67fc4664dd6bd4e6335b3/libmscore/note.cpp#L384) or [here](https://github.com/mattmcclinch/MuseScore/blob/bd64822d8fb3e7f18dd67fc4664dd6bd4e6335b3/libmscore/note.cpp#L397).